packages/account/src/account.ts

```diff
@@ -151,7 +151,7 @@ export class Account extends AbstractAccount {
   async getCoins(assetId?: BytesLike): Promise<Coin[]> {
     const coins = [];
 
-    const pageSize = 9999;
+    const pageSize = 100;
```

Comment:
@Torres-ssf I believe this will all change after #2408, but I had to lower the limits to conform to the new complexity limits. I arbitrarily used 100 to test it out.